robotiq_c_model_control/src/robotiq_c_model_control/robotiq_gripper_action_interface.py

```diff
@@ -274,50 +348,42 @@ def reset(self):
         goal = CModel_robot_output()
         goal.rACT = 1
         self.send_gripper_command(goal, parse=False)
+
+        # Wait for gripper to activate
+        timer = RobotiqCommandTimeout(seconds=3.0)
+        timer.start()
         while not rospy.is_shutdown() and not self.interrupted:
             with self.status_cv:
                 self.status_cv.wait(0.25)
                 status = deepcopy(self.last_status)
             if status.gSTA == 3:
                 break
+            if timer.expired():
+                self.__log.err('Timeout on activate w/ status: "{}"'.format(self.grip_status_to_str(status)))
```

Comment:
should this send a banner message to chiron?

Reply:
So, self.__log.banner in favor of self.__log.err?